python/catboostserver/Makefile

```
@@ -0,0 +1,17 @@
+dev_install:
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)